rhasspyclient/__init__.py

```diff
@@ -25,10 +25,15 @@
 class RhasspyClient:
     """Client object for remote Rhasspy server."""
 
-    def __init__(self, api_url: str, session: aiohttp.ClientSession):
-        self.api_url = api_url
-        if not self.api_url.endswith("/"):
-            self.api_url += "/"
+    def __init__(
+        self, HOST: str, PORT: str, session: aiohttp.ClientSession, secure: bool = False
+    ):
+        if secure:
+            self.api_url = "https://{}:{}/api/".format(HOST, PORT)
+            self.events_url = "wss://{}:{}/api/events/".format(HOST, PORT)
+        else:
+            self.api_url = "http://{}:{}/api/".format(HOST, PORT)
+            self.events_url = "ws://{}:{}/api/events/".format(HOST, PORT)
```

Comment:
I wouldn't use uppercase parameter names `HOST` and `PORT`, as the rest of the code always uses lowercase parameter names.

Reply:
Gotcha, will change